cli/cmd/login.go

```diff
@@ -11,6 +11,7 @@ import (
 	"os/signal"
 	"strconv"
 	"strings"
+	gosync "sync"
```

Comment:
`gosync`, why? is there another package or variable with this name, or just editor mishap?

Reply:
There's another sync package